src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -322,6 +322,7 @@
       <_ILRepackArgs>/out:&quot;$(MSBuildThisFileDirectory)$(IntermediateOutputPath)$(AssemblyName).dll&quot; /internalize</_ILRepackArgs>
       <_ILRepackArgs>$(_ILRepackArgs) &quot;$(MSBuildThisFileDirectory)$(IntermediateOutputPath)$(AssemblyName).dll&quot;</_ILRepackArgs>
       <_ILRepackArgs>$(_ILRepackArgs) @(_InputAssembliesThatExist->'&quot;%(Identity)&quot;', ' ')</_ILRepackArgs>
+      <_ILRepackArgs>$(_ILRepackArgs) /lib:&quot;$(XAInstallPrefix)xbuild-frameworks\MonoAndroid\v1.0\facades&quot;</_ILRepackArgs> <!-- Path to netstandard.dll -->
```

Comment:
I'm wondering if this is right... `netstandard.dll` in this case is coming from our build tree, and meant for Xamarin.Android projects. Xamarin.Android.Build.Tasks is a desktop assembly, should it be using Mono's system install of `netstandard.dll` instead?

Reply:
We were previously repacking against the netstandard.dll that was copied into this directory by some Java.Interop project. Maybe we can do something similar to what macios is doing here... https://github.com/xamarin/xamarin-macios/blob/0557f16e6a936f9d3c94fe020536141296b42640/msbuild/ILMerge.targets#L13

Reply:
I just rebuilt on macOS with the latest changes, and we should now using a path from the .NET Core installation:

> /lib:"/usr/local/share/dotnet/sdk/NuGetFallbackFolder/netstandard.library/2.0.3/build/netstandard2.0/ref/"

Reply:
Looks like Windows wasn't happy with this approach